packages/reports/addon/components/navi-request-column-config/metric.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Description: Navi Request Column Config Metric Component
+ *
+ * Usage:
+ *  <NaviRequestColumnConfig::Metric
+ *    @column={{editingColumn}}
+ *    @metadata={{visualization.metadata}}
+ *    @onClose={{action "onClose"}}
+ *    @onUpdateColumnName={{action "onUpdateColumnName"}}
+ *  />
+ */
+import Component from '@ember/component';
+import { layout as templateLayout, tagName } from '@ember-decorators/component';
+import layout from '../../templates/components/navi-request-column-config/metric';
+import { action, set, computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+@tagName('')
+@templateLayout(layout)
+class Metric extends Component {
+  /**
+   * @service
+   */
+  @service('metric-parameter') parameterService;
+
+  /**
+   * @override
+   * @method init
+   */
+  init() {
+    super.init(...arguments);
+    this._previousMetric = null;
+    this._parametersPromise = null;
+  }
+
+  /**
+   * @override
+   * @method didReceiveAttrs
+   */
+  didReceiveAttrs() {
+    super.didUpdateAttrs(...arguments);
```

Comment:
`this._super(...arguments);` ?

Reply:
Nope this is the right way for native classes, but it was the wrong method 😮

Reply:
lol yeah I updated the comment a second later